pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -817,13 +817,15 @@
         "1.1.37",
         "1.2.0",
         "1.1.36",
-        "1.3.0"
+        "1.3.0",
+        "1.3.1"
       ],
-      "BaselineVersion": "1.3.0",
+      "BaselineVersion": "1.3.1",
```

Comment:
It was intentional, as the new package will have optimization data, which would, in theory, help everyone who uses this.